iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/SubstationXml.java

```diff
@@ -44,11 +45,6 @@ protected void writeRootElementAttributes(Substation s, Network n, NetworkXmlWri
         if (s.getTso() != null) {
             context.getWriter().writeAttribute("tso", context.getAnonymizer().anonymizeString(s.getTso()));
         }
-        if (!s.getGeographicalTags().isEmpty()) {
-            context.getWriter().writeAttribute("geographicalTags", s.getGeographicalTags().stream()
-                    .map(tag -> context.getAnonymizer().anonymizeString(tag))
-                    .collect(Collectors.joining(",")));
-        }
```

Comment:
Why don't we use `IidmXmlUtil.runUntilMaximumVersion` like in `readRootElementAttributes` to write the geographicalTags in previous iidm versions?

Reply:
That was made in the first commit, but @MioRtia expect from me to change for this.

>  MioRtia on 23 Jun Member
> 
> Don't use that, simply delete this part: the geographical tags will be written as properties in any case if they are defined. > However, you will need to change the read method: until 1.2, you can read geographicalTags and set them as properties.